jenkins-pipelines/artifacts-centos7.jenkinsfile

```diff
@@ -7,7 +7,7 @@ artifactsPipeline(
     test_config: 'test-cases/artifacts/centos7.yaml',
```

Comment:
looks a bit confusing like there's a duplicate definition both in this pipeline and the yaml inside..

Reply:
wdym? I removed the definition in the yaml